package.json

```diff
@@ -22,6 +22,8 @@
     "release": "changeset publish",
     "preinstall": "git config --global url.\"https://github.com/\".insteadOf git://github.com/",
     "build": "lerna run --scope @razorpay/blade build",
+    "test:web": "yarn workspace @razorpay/blade test:react",
+    "test:native": "yarn workspace @razorpay/blade test:react-native",
     "start:web": "yarn workspace @razorpay/blade react:storybook",
     "start:ios": "yarn workspace @razorpay/blade react-native:storybook:ios",
     "start:android": "yarn workspace @razorpay/blade react-native:storybook:android"
```

Comment:
I'm sick of writing `yarn workspace @razorpay/blade` every time and then remember all the permutations of 

```ts
"test:react:ios" | "test:react:android" | "test:react:start" | "test:react:build" | "test:react-native:ios" | "test:react-native:android" | "test:react-native:start" | "test:react-native:build"
```

Reply:
but why don't you make your current working directory as `packages/blade`?

Reply:
also install https://fig.io/ and you won't have to remember all the P&C you'll get intellisense in your teminal

Reply:
Why have monorepo in that case? 🤔 If i'm going to CD into it anyways. Like won't installing deps without specifying the workspace mess things up?

Reply:
it's a monorepo since we wanted to have `old-blade` in this repo itself. it's just that we are sharing the eslint, tsconfig and few other things via this setup(debt we want to remove). and CD you can just do once. Also installing dependencies won't create mess since it won't let you install dependencies without mentioning the workspace name(it'll just fail, you can try out)